core/templates/domain/collection/collection-rights-backend-api.service.ts

```diff
@@ -16,143 +16,168 @@
  * @fileoverview Service to change the rights of collections in the backend.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
-
-angular.module('oppia').factory('CollectionRightsBackendApiService', [
-  '$http', '$q', 'UrlInterpolationService',
-  'COLLECTION_RIGHTS_URL_TEMPLATE',
-  function($http, $q, UrlInterpolationService,
-      COLLECTION_RIGHTS_URL_TEMPLATE) {
-    // Maps previously loaded collection rights to their IDs.
-    var collectionRightsCache = {};
-
-    var _fetchCollectionRights = function(collectionId, successCallback,
-        errorCallback) {
-      var collectionRightsUrl = UrlInterpolationService.interpolateUrl(
-        COLLECTION_RIGHTS_URL_TEMPLATE, {
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { CollectionEditorPageConstants } from
+  'pages/collection-editor-page/collection-editor-page.constants';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Move cloneDeep above all Oppia dependencies. Also, please alphabetise the imports.

Reply:
So utilities, in general, should be placed above all other imports?

Reply:
```
import { downgrade }..
import { HttpClient } ...

import cloneDeep ...

import oppia dependecies...
```

Reply:
Done!